controllers/helmchart_controller_test.go

```diff
@@ -65,7 +67,8 @@ func TestHelmChartReconciler_Reconcile(t *testing.T) {
 	g.Expect(err).NotTo(HaveOccurred())
 	defer os.RemoveAll(server.Root())
 
-	g.Expect(server.PackageChartWithVersion(chartPath, chartVersion)).To(Succeed())
+	publicKeyPath := fmt.Sprintf("%s/%s", server.Root(), publicKeyFileName)
+	g.Expect(server.PackageSignedChartWithVersion(chartPath, chartVersion, publicKeyPath)).To(Succeed())
```

Comment:
Maybe it'd be better to create a copy of this test and have integrity check in that only. I think we'd like to test without integrity check as well.